src/dotty/tools/dotc/core/Symbols.scala

```diff
@@ -417,14 +417,6 @@ object Symbols {
     /** If this symbol satisfies predicate `p` this symbol, otherwise `NoSymbol` */
     def filter(p: Symbol => Boolean): Symbol = if (p(this)) this else NoSymbol
 
-    /** Is this symbol a user-defined value class? */
-    final def isDerivedValueClass(implicit ctx: Context): Boolean = {
```

Comment:
`sym.isDerivedValueClass` feels more preferable to me than `isDerivedValueClass(sym)`